### ISSUE TITLE
Few-shot Knowledge Transfer for Fine-grained Cartoon Face Generation

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#37]
# TL;DR
GAN to convert a face image into an animated one. The strategy is that at first training the basic model using a broad domain (e.g., women in general),  then fine tuned in to maller domains (e.g., old people, children, young people, etc.). Use auxiliary classifier to reflect the differences in each domain. The generated images seem to be able to capture the unique characteristics of each person.
![image](https://user-images.githubusercontent.com/24524018/89734150-4755a980-da95-11ea-84db-264404d5bdd0.png)
![image](https://user-images.githubusercontent.com/24524018/89734152-4b81c700-da95-11ea-857e-df5fa61953ed.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.13332

# Submission Dates(yyyy/mm/dd)
2020/07/27

# Authors and institutions
Nan Zhuang, Cheng Yang
    - Peking University, Beijing, China 
    - ByteDance AI Lab, Beijing, China

# Methods

# Results
![image](https://user-images.githubusercontent.com/24524018/89734155-4f154e00-da95-11ea-87d9-4db44ca2fe21.png)

# Comments
